### PR TITLE
Plugin silex

### PR DESCRIPTION
**Pull request by henrikbjorn**

### CODE REVIEW

**Comment by beberlei** [comment]
docs?

**Comment by henrikbjorn** [comment]
Now with documentation.

File: README.md [hunk 1]
```diff
@@ -243,9 +243,9 @@ the EntityManager:
 
 ### Silex
 
-Silex plugin ships with a CommandBus that knows how to get services out of your Silex application
-aswell as a ServiceProvider. The ServiceProvider adds the most basic services to get LiteCQRS to
-run.
+Silex plugin ships with a CommandBus and a EventMessageBus that knows how to get services out of
+your Silex application asw ell as a ServiceProvider. The ServiceProvider adds the most basic services
```
Comment (stof) [inline README.md]: typo here (missplaced space)

**Comment by henrikbjorn** [comment]
@beberlei should be ready to :ship:

**Pull request merged**